### ISSUE TITLE
Update the `builds.json` format in s3

### ISSUE THREAD

**Issue by gursewak1997** [coreos/fedora-coreos-pipeline#1042]
After  the initial round of garbage collection, we updated the `builds.json` for Rawhide and Branched. We pruned the cloud-uploads and added the following to the respective build info in the JSON file.
```
{
   "policy-cleanup": [
         "cloud-uploads"
   ]
},
```
Now, we need to update the `policy-cleanup` type from a list to a dict if we are looking to add the `images-kept` to it to have something like this.
```
// Valid JSON
{
    "policy-cleanup": {
        "cloud-uploads": True,
        "images": True
        "images-kept": ["qemu", "live-iso"] 
    }
}
```

To get to that, let's first have the `builds.json` manually changed for those two streams.
Basically changing the `"cloud-uploads"` to `"cloud-uploads": true`

**Comment by gursewak1997** [comment]
Updated the files.
